routes/api/users.js

```diff
@@ -1,6 +1,9 @@
 const router = require("express").Router();
 const passport = require("passport");
 const User = require("../../models/").User;
+const TokenService = require('../../services/TokenService');
+const dotenv = require('dotenv/config');
+const AuthValidator = require('../../validators/AuthValidator');
```

Comment:
put the validator in a middlewares folder

Reply:
I actually synced with my teammates and found that someone else was working on validator so i removed it